client-side-js/testLibrary.js

```diff
@@ -10,7 +10,13 @@ async function initOPA(pageObjectConfig, browserInstance) {
                         pageConfig[pageKey] = new window.fe_bridge[className](options)
                     })
                 })
+
                 sap.ui.test.Opa5.createPageObjects(pageConfig)
+                // use the same timouts and intervals that wdi5 uses
+                sap.ui.test.Opa.extendConfig({
+                    timeout: new Date(window.wdi5.waitForUI5Options.timeout).getSeconds(), // convert milliseconds to seconds
+                    pollingInterval: window.wdi5.waitForUI5Options.interval
```

Comment:
same q as above - might there be a potential runtime error for an `undefined` config option?

Reply:
same answer as above